dysnix/bsc/values.yaml

```diff
@@ -155,11 +158,11 @@ ingress:
 
 ## Autoscaling parameters
 autoscaling:
-  enabled: true
-  scaleDownDisabled: true
-  targetCPU: 60
-  minReplicas: 1
-  maxReplicas: 10
+  enabled: false
```

Comment:
For some reason kind cluster, which is used in CI, stopped providing [metrics](https://github.com/dysnix/charts/runs/3221102130?check_suite_focus=true#step:9:53) api. That's why default `values.yaml` was disabled in commented.